hydra/_internal/deprecation_warning.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import os
+import warnings
+
+from hydra.errors import HydraDeprecationError
+
+
+def deprecation_warning(message: str, stacklevel: int = 1) -> None:
+    warnings_as_errors = os.environ.get("HYDRA_DEPRECATION_WARNINGS_AS_ERRORS", None)
```

Comment:
nit: dict.get() returns None by default for non-existing keys.

This should be equivalent:
```suggestion
    warnings_as_errors = os.environ.get("HYDRA_DEPRECATION_WARNINGS_AS_ERRORS")
```

Reply:
Oh, nice! I think I was confusing `dict.get` with `dict.pop`, which would raise an error if no default is supplied.